services/QuillLessonsServer/bootstrap.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+echo "Installing script dependencies..."
+brew install jq
```

Comment:
Since there is only one short line of JSON, I don't think you need this dependency. Something like this would have the same effect (using single quotes to get in/out of calling variables):
```bash
CERT="some string"
JSON=\''{"env": {"JWT_PUBLIC_KEY": "'"$CERT"'"}}'\'
```

Reply:
I've spent about an hour trying to figure out the best way to handle formatting in this approach.  My own bash is rusty enough that I still haven't found a good way to convert newlines to `\n`, which is needed to put the data in JSON format.  Thoughts?

Reply:
My first instinct would be to use `sed`. It does seem like the proper format for doing this correctly (option [here](https://stackoverflow.com/questions/50482326/how-to-replace-string-with-certificate-string-in-json-file-shell) and [here](https://stackoverflow.com/questions/38672680/replace-newlines-with-literal-n)) are a little hard to parse. 

It's not worth spending any more time on it, you can use `jq` if you have that working at this point.

Reply:
Yeah, I tried some of that sed stuff after googling, but couldn't get it to work, and don't really understand enough about the stuff that's going on in there with the sed iterator to figure out what I was doing wrong.  Guess I'll just leave this as-is for now, and keep the dependency.